.circleci/config.yml

```diff
@@ -0,0 +1,104 @@
+---
+version: 2.1
+
+orbs:
+  prometheus: prometheus/prometheus@0.1.0
+
+executors:
+  # Whenever the Go version is updated here, .promu.yml should
+  # also be updated.
+  golang:
+    docker:
+    - image: circleci/golang:1.13
+
+jobs:
+  test:
+    executor: golang
+
+    steps:
+    - prometheus/setup_environment
+    - run: make
+    - prometheus/store_artifact:
+        file: bind_exporter
+
+  codespell:
+    docker:
+    - image: circleci/python
+
+    steps:
+    - checkout
+    - run: sudo pip install codespell
+    - run: codespell --skip=".git,./vendor,ttar,go.mod,go.sum,*pem" -L uint,packages\',uptodate
+
+  build:
```

Comment:
Can't you use the `build` job from the CircleCI's orb as [here](https://github.com/prometheus/prometheus/blob/000e306f2fbb373bc37fa8e710553b072fac71a1/.circleci/config.yml#L97-L101)?

Reply:
Sure, again, copy-pasta from node_exporter.